pom.xml

```diff
@@ -329,11 +333,6 @@
 			<artifactId>micrometer-core</artifactId>
 			<version>${micrometer.version}</version>
 		</dependency>
-		<dependency>
-			<groupId>io.micrometer</groupId>
-			<artifactId>micrometer-registry-prometheus</artifactId>
-			<version>${micrometer.version}</version>
-		</dependency>
```

Comment:
Does it work if we avoid to remove this dependency for now? I am still leaving ALL dependencies for the current bridge even if they are not used in the Quarkus version. In the end, a clean up will be needed.

Reply:
I tried it and I was getting error due to it but I can try again